simple_network_sim/network_of_populations.py

```diff
@@ -83,10 +100,26 @@ def basicSimulationInternalAgeStructure(network, timeHorizon):
 
 # CurrentlyInUse
 def totalIndividuals(nodeState):
+    """Calculate the total population size over all ages and regions.
+
+    :param nodeState: The disease status of the population stratified by age.
```

Comment:
I think you should at least mention that this is the data from a node (or region).